.coveragerc

```diff
@@ -3,10 +3,16 @@ source = hip_data_tools
 
 [report]
 exclude_lines =
-    if self.debug:
     pragma: no cover
+    def __repr__
+    if self.debug:
+    if settings.DEBUG
+    raise AssertionError
     raise NotImplementedError
+    if 0:
     if __name__ == .__main__.:
+    class .*\bProtocol\):
```

Comment:
Curious why including this?

Reply:
I forgot this repo uses a package called semantic release to auto-release. The last PR i did not use fix: or feat: in the title as i did not change any .py only requirements.txt

See the PR notes

I need a non-breaking change like this one along with a feat: or fix: title so i can make sure the changes from the last PR are released as a new build and onto pypi as this is a public repo and used by our airflow instance to pull this package in for utilities.